Gopkg.lock

```diff
@@ -1270,14 +1259,6 @@
   revision = "4a180b209f5f494e5923cfce81ea30ba23915877"
   version = "v2.18.06"
 
-[[projects]]
-  branch = "master"
-  digest = "1:99c6a6dab47067c9b898e8c8b13d130c6ab4ffbcc4b7cc6236c2cd0b1e344f5b"
-  name = "github.com/shirou/w32"
-  packages = ["."]
-  pruneopts = "UT"
-  revision = "bb4de0191aa41b5507caa14b0650cdbddcd9280b"
-
```

Comment:
Um. This should not have gotten deleted as far as I can tell. Looking into it.

Reply:
Thanks and apologies.

Reply:
Not your fault at all! That frustration was directed at dep.

Reply:
Well, I could've seen something unexpected was happening. :-)